File: FoodieAppBE/SellerServiceApp/src/main/java/com/project/SellerServiceApp/domain/Restaurant.java
package com.project.SellerServiceApp.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import javax.persistence.Lob;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
@Document(collection = "restaurantService")
public class Restaurant
{
    @Id
    private String restaurantId;
    private  String restaurantName;
    @Lob
    private byte[] restaurantImage;
    private List<Cuisines> menu;
    private Address address;
    private String sellerEmail;
    private List<Order> orders;
}

File: FoodieAppFE/src/app/model/Seller/Restaurant.ts
import { Order } from "../Order";
import { cuisines } from "./Cuisines";
import { SAddress } from "./SAddress";

export class Restaurant{
    restaurantId?:string;
    restaurantName?:string;
     restaurantImage?:any[];
  menu?:cuisines[];
     address?:SAddress;
     sellerEmail?:string;
    orders?:Order[]
}